src/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,56 @@
+import classNames from 'classnames';
+import { useState } from 'react';
+
+type Props = {
+  setFilter: (filter?: boolean) => void
+};
+
+enum TypeFilter {
+  'all',
+  'active',
+  'completed',
+}
+
+export const Filter: React.FC<Props> = ({ setFilter }) => {
+  const [typeFilter, setTypeFilter] = useState(TypeFilter[1]);
+
+  const chooseFilter = (type: string, filter?: boolean) => {
+    setFilter(filter);
```

Comment:
Why do you need it? `setFilter(filter)` 
Seems that it's redundant a bit

Reply:
here i use for render new list todos, where change type filter